vault-granular-access-control/index.json

```
@@ -0,0 +1,106 @@
+{
+  "pathwayTitle": "Learn HashiCorp Vault",
+  "title": "Granular Access Control with Vault Policies",
+  "description": "Teaches you how to create policies that are secure, flexible, and reusable for your team.",
+  "difficulty": "beginner",
+  "time": "20 minutes",
+  "icon": "fa-vault",
+  "details": {
+    "intro": {
+      "text": "intro.md",
+      "credits": "Learn more at https://learn.hashicorp.com/vault"
+    },
+    "steps": [
+      {
+        "title": "Your environment",
+        "text": "step0.md",
+        "courseData": "step0.setup.sh"
+      },
+      {
+        "title": "The app client",
+        "text": "step-app-intro.md",
+        "courseData": "step-app-intro.sh"
+      },
+      {
+        "title": "App requirement #1",
+        "text": "step-app-1.md"
+      },
+      {
+        "title": "App answer #1",
+        "text": "step-app-1-answer.md"
+      },
+      {
+        "title": "App requirement #2",
+        "text": "step-app-2.md"
+      },
+      {
+        "title": "App answer #2",
+        "text": "step-app-2-answer.md"
+      },
+      {
+        "title": "App requirement #3",
+        "text": "step-app-3.md"
+      },
+      {
+        "title": "App answer #3",
+        "text": "step-app-3-answer.md"
+      },
+      {
+        "title": "The admin client",
+        "text": "step-admin-intro.md",
+        "courseData": "step-admin-intro.sh"
+      },
+      {
+        "title": "Admin requirement #1",
+        "text": "step-admin-1.md"
+      },
+      {
+        "title": "Admin answer #1",
+        "text": "step-admin-1-answer.md"
+      }
+    ],
+    "finish": {
+      "text": "finish.md",
+      "credits": "Learn more at https://learn.hashicorp.com/vault"
+    },
+    "assets": {
+      "host01": [
+        {
+          "file": "admins-policy.hcl",
+          "target": "~/"
+        },
+        {
+          "file": "readonly.sql",
+          "target": "~/"
+        },
+        {
+          "file": "apps-policy.hcl",
+          "target": "~/"
+        }
+      ]
+    }
+  },
+  "environment": {
+      "showdashboard": true,
+      "uilayout": "terminal",
```

Maybe use Katacoda text editor to make it easier to write policies???

```suggestion
      "uilayout": "editor-terminal",
```

---

I tend to hate how much the file editor takes up but this the right move for ease-of-use.